build/ios/OWNERS

```diff
@@ -1,4 +1,2 @@
 rohitrao@chromium.org
 stuartmorgan@chromium.org
```

Comment:
🤣 

I'm pretty sure Rohit and I don't need to be owners of this directory in the Flutter buildroot, so you could just remove this file. (And, presumably, any other OWNERS files we have lying around, since (a) they will all be wrong, and (b) we don't use OWNERS AFAIK.)

Reply:
deleted all owners files